Makefile

```diff
@@ -16,7 +16,7 @@ coverage:
 
 check-style:
 	@# Keep the version in sync with the command in .circleci/config.yml
-	$(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.46.2
+	$(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.49.0
```

Comment:
iirc, this is included in `cimg/go`? (maybe not that exact version though)

Reply:
It might. I have another ticket (https://mattermost.atlassian.net/browse/MM-50828) to migrate the CI pipeline to GH actions. That will be a great opportunity to take a holistic look at CI.